models/account.js

```diff
@@ -46,7 +46,7 @@ var AccountSchema = new Schema({
 	},
 	rank: {
 		type: String,
-		enum: require('../config/userRights').roles,
+		enum: require('../public/dist/js/userRights').roles,
```

Comment:
but if there's no other way... then we'll have to live with this new type of requires.